app/templates/application.hbs

```diff
@@ -1,4 +1,4 @@
-{{new-osf-navbar hostAppName='Preprints' signupUrl=theme.signupUrl loginAction=(action 'login')}}
+{{new-osf-navbar hostAppName='Preprints' signupUrl=theme.signupUrl loginAction=(action 'login') pathPrefix=theme.pathPrefix}}
```

Comment:
`pathPrefix` is a very generic name -- it might be clearer (and more consistent with `signupUrl`) to pass the whole submit url as `addPreprintUrl` or something like that.